q2_longitudinal/_utilities.py

```diff
@@ -240,7 +240,7 @@ def _compare_pairwise_differences(groups, parametric=False):
                 t, p = wilcoxon(values)
         except ValueError:
             # if test fails (e.g., because of zero variance), just skip
```

Comment:
Drive-by Comment:

Don't forget to update or remove this comment.

Reply:
I left in intentionally... I see assigning `np.nan` as still skipping, just in a more functional way, so I think this comment still applies but happy to update if you have a suggestion.

Reply:
Oh I think I see now, although that wasn't clear to me at first. How about restructuring a bit? Revert back to the `pass` here (keep the comment as-is), and move the `t = p = np.nan` business up just before the `try` statement. This proposed approach handles the null-case straight off, which I think has a few benefits.